src/lib/rpcWallet.js

```diff
@@ -291,8 +291,6 @@ export default {
       browser.tabs.reload(aepp.connection.port.sender.tab.id);
       this.sdk.removeRpcClient(aepp.id);
     });
-    delete walletController.wallet;
-    this.initFields();
   },
   async [AEX2_METHODS.INIT_RPC_WALLET]({ address, network }) {
```

Comment:
Can we have a consistent function naming convention in this file?  Quoting this change: `async [AEX2_METHODS.LOGOUT]()` -> `async disconnect()`

Reply:
I assuming `AEX2_METHODS` a deprecated object (mainly because it is not connected to AEX2, but message exchanging between background and popup) and planning to get rid of it in future PRs.